rental_class.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
```

Comment:
- [ ] Kindly include `book` and `person` in your initialize method so that it can reflect the `many-to-many` relationship. 👍